pkg/storage/juicefs.go

```diff
@@ -122,13 +123,29 @@ func (s *JuiceFsStorage) Format(fsName string) error {
 }
 
 func (s *JuiceFsStorage) Unmount(localPath string) error {
-	cmd := exec.Command("juicefs", "umount", "--force", localPath)
+	juiceFsUmount := func() error {
+		cmd := exec.Command("juicefs", "umount", localPath)
 
-	output, err := cmd.CombinedOutput()
+		output, err := cmd.CombinedOutput()
+		if err != nil {
+			log.Printf("error executing juicefs umount: %v, output: %s", err, string(output))
```

Comment:
Can you clean up this log

Reply:
it's the same log as [here](https://github.com/beam-cloud/beta9/pull/402/files#diff-c6dd2c832240e75f3b4b734ef955e8cfeba0121667164d22f6bc10da2e881118L129)